.github/CODEOWNERS

```diff
@@ -173,7 +173,7 @@
 #/<NotInRepo>/            @ms-premp @qiaozha
 
 # PRLabel: %Communication
-/sdk/communication/                             @RezaJooyandeh @DominikMe @ankitarorabit @Azure/azure-sdk-communication-code-reviewers
+/sdk/communication/                             @acsdevx-msft
```

Comment:
@meeraharidasa @RezaJooyandeh can you help me understand the goal/intend with switching to this account? It cause some things like pipeline notifications to be invalid because we have not good way to map that alias to an AAD account we can configure our DevOps pipeline notifications.

Reply:
Also it will not end up adding any any individuals as pull request owners when files under communication are touched.